src/app/routes/assets/verification-method/verification-method.component.ts

```diff
@@ -22,8 +22,15 @@ export class VerificationMethodComponent implements OnInit {
   verificationsAmount: number;
   dataSource: PublicKey[] = [];
   selectControl = new FormControl('', [Validators.required]);
+
+  publicKeyExistValidator(): ValidatorFn {
```

Comment:
Method shouldn't be mixed with properties. Property > constructor > methods. Please align that.